### PR TITLE
ansible: update rackspace windows VMs in inventory

### PR DESCRIPTION
**Pull request by StefanStojanovic**
This PR finalizes cleaning the Rackspace Windows portion of the inventory.yml file. It removes the deleted machines and adds newly created ones.

After landing this, I'll make the necessary changes in the secrets repo to follow these ones.

Refs: https://github.com/nodejs/build/issues/3373

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
Rubber Stamp LGTM

File: ansible/inventory.yml [hunk 1]
```diff
@@ -324,9 +324,16 @@ hosts:
...
+        win2019_vs2019-x64-1: {}
+        win2019_vs2019-x64-2: {}
+        win2019_vs2019-x64-3: {}
+        win2019_vs2019-x64-4: {}
+        win2022_vs2022-x64-1: {}
+        win2022_vs2022-x64-2: {}
+        win2022_vs2022-x64-3: {}
+        win2022_vs2022-x64-4: {}
+        win2022_vs2022-x64-5: {}
+        win2022_vs2022-x64-6: {}
```
Comment (richardlau) [inline ansible/inventory.yml:336]: Aren't these missing the IP address?

Comment (StefanStojanovic) [reply ansible/inventory.yml:336]: I planned to add them to the secrets repo as an additional safety measurement, but if they should be here for consistency with other machines or any other reason, I'll add them.

Comment (richardlau) [reply ansible/inventory.yml:336]: Adding them to the secrets repo is okay -- as long as they're added somewhere 🙂.

**Review by richardlau — APPROVED** [review]

**Pull request merged**